src/runtime/mod.ts

```diff
@@ -3,7 +3,10 @@ import { cd } from "./cd.ts";
 import { exec } from "./exec.ts";
 import { quote } from "./quote.ts";
 
-export type $ = typeof exec & typeof colors & {
+export { ProcessError } from "./process_error.ts";
+export { ProcessOutput } from "./process_output.ts";
+
+export type $Global = typeof exec & typeof colors & {
```

Comment:
The deno language server was complaining about a duplicate identifier (I believe because there was both a `const` and a `type` in scope called `$`. Changing the `type` to *anything else* fixes this - I labored over what to rename it to so as to be minimally deviant from your original pattern.

I don't think this is something that's ultimately too consequential, so changing back is fine by me - I'll put it back in a few minutes :+1:

Reply:
ah yeah now i can remember, but than we can simple import the `$` symbol with an alias like i'm doing this with the other imports in `types.d.ts`, than we don't have to rename the global type.
```diff
import type {
- $,
+ $ as _$,
  async as _async,
```

Reply:
that does work too, and is what I did initially - as it is now (after my last commit which changed it back to just `$`), vscode and the language server are actually not complaining.... :man_shrugging: no idea why not, but I think nothing else is actually needed to appease the red-squiggly-underline overlords :)

Reply:
yeah had the same issue in a local branch where i aliased it to `_$` but currently it seems to work fine for me too 👍